src/library/dlgcoverartfullsize.cpp

```diff
@@ -144,22 +163,27 @@ void DlgCoverArtFullSize::slotCoverFound(
     // whitespace appearing on the side when resizing a window whose
     // borders touch the edges of the screen.
     QSize dialogSize = m_pixmap.size();
+    QWidget* parent = parentWidget();
+    VERIFY_OR_DEBUG_ASSERT(parent) {
+        qWarning() << "DlgCoverArtFullSize does not have a parent.";
+        parent = this;
```

Comment:
The name of the local variable is misleading when reusing it for this. If you don't know a better name then just use `parentOrThis`.